ingest/bin/calculate-gene-converage-from-nextclade-translation.py

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env python3
+
+import argparse
+from sys import stderr
+
+from Bio import SeqIO
+import re
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Calculate gene coverage from amino acid sequence",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser.add_argument(
+        "--fasta",
+        type=str,
+        help="FASTA file of CDS translations from Nextclade.",
+    )
+    parser.add_argument(
+        "--out-col",
+        type=str,
+        default="gene_coverage",
+        help="Output column name.",
+    )
+    return parser.parse_args()
+
+
+def calculate_gene_coverage_from_nextclade_cds(fasta, out_col):
+    """
+    Calculate gene coverage from amino acid sequence in gene translation FASTA file from Nextclade.
+    """
+    print(f"genbank_accession\t{out_col}")
+    # Iterate over the sequences in the FASTA file
+    for record in SeqIO.parse(fasta, "fasta"):
+        sequence_id = record.id
+        sequence = str(record.seq)
+        
+        # Calculate gene coverage
+        results = re.findall(r"([ACDEFGHIKLMNPQRSTVWY])",  sequence.upper())
```

Comment:
I see it. There's definitely a trade-off of hardcoding the amino acids (20 letters) explicitly here which takes 1 code line verses importing a whole library (BioPython) just to use the same 20 letters and expose ourselves to the risk that BioPython changes the API to access those 20 letters.

Feel free to submit a commit suggestion though! I'm happy to commit a change if someone else codes it.

If we were additionally using other methods from BioPython, the cost-to-benefit ratio would be different. Ditto if we somehow "discover novel amino acids" and the letters expand (["Frozen, but no accident - why the 20 standard amino acids were selected" by Doig 2017](https://pubmed.ncbi.nlm.nih.gov/27926995/)).

Reply:
We are already using Bio.SeqIO here, so I thought would make sense to look in BioPython. Now that I'm thinking about it again, we really care about what [Nextclade AA letters](https://github.com/nextstrain/nextclade/blob/c9d43865f7df492a3c2a6b2e1a9287c670155fd9/packages/nextclade/src/translate/translate.rs#L8-L187) are since we are reading in the Nextclade translations. 

But again, not asking for a change!